app/policies/user_policy.rb

```diff
@@ -10,17 +10,15 @@ def edit_password?
     @current_user == @user
   end
 
-  def edit_roles?
-    (@current_user.data_coordinator? || @current_user.support?) && @user.active?
-  end
-
   %w[
     edit_roles?
     edit_dpo?
     edit_key_contact?
   ].each do |method_name|
     define_method method_name do
-      (@current_user.data_coordinator? || @current_user.support?) && @user.active?
+      return true if @current_user.support?
```

Comment:
Does active not matter for support? (Actually, does it matter for anyone since you can't log in while not active?)

Reply:
Oh, so I don't think it matters for current user, cause as you say, they should be able to login before they even get to this point. But for the user that is getting edited we have an AC to allow deactivated users to be moved. So I'm allowing other fields for deactivated user to be editable too (cause otherwise we'd only show organisation question on the edit page, which is a bit odd that support can change the org but nothing else)

So basically support users can edit deactivated and active users, whereas coordinators can only edit active as it was before.

Hope this makes sense